components/buttongroup/metadata/buttongroup.yml

```diff
@@ -30,3 +51,20 @@ examples:
           <span class="spectrum-Button-label">Rate now</span>
         </button>
       </div>
+
+  - id: buttongroup-vertical-small
+    name: Vertical - Small
+    description: |
+      Spacing for the Small t-shirt size.
+    markup: |
+      <div class="spectrum-ButtonGroup--sizeS spectrum-ButtonGroup--vertical">
```

Comment:
line 60 should be 
`<div class="spectrum-ButtonGroup spectrum-ButtonGroup--sizeS spectrum-ButtonGroup--vertical">`

(missing spectrum-ButtonGroup base class)